scripts/dashboard/medperf_dashboard/get_data.py

```diff
@@ -0,0 +1,126 @@
+import os
+import pandas as pd
+import datetime
+
+from medperf.entities.dataset import Dataset
+from medperf import config
+
+from .utils import get_institution_from_email, get_reports_path, stage_id2name
+
+from typer import Typer, Option, run
+
+app = Typer()
+
+
+def get_dsets(mlcube_id):
+    dsets = Dataset.all(filters={"mlcube": mlcube_id})
```

Comment:
I cannot get quite well how does this work? `Dataset.all` uses `config.comms` Object but it is properly initialized only `medperf/cli/medperf/init.py:initialize` that doesn't seem to be called. I don't have a properly set up env but it looks like `medperf-dashboard` works (I mean, `config.comms` is initialized and really tries to reach the server)